README.md

```diff
@@ -164,18 +179,34 @@ AZURE_CLIENT_SECRET
 After following the steps in [Development](#dev), you can run the tests:
 
 ```bash
-$ pytest -n 10 .
+pytest .
+```
+
+To create an interface test in your robot repository, use the function `interface_test` from `robot_interface`. The
+argument should be an interface object from your robot specific implementation.
+See [isar-robot](https://github.com/equinor/isar-robot/blob/main/tests/interfaces/test_robotinterface.py) for example.
+
+### Integration tests
+
+Integration tests can be found [here](https://github.com/equinor/isar/tree/main/tests/integration) and have been created
+with a simulator in mind. The integration tests will not run as part of `pytest .` or as part of the CI/CD pipeline. To
+run the integration tests please follow the instructions in blabla for setting up the `isar-turtlebot` implementation
```

Comment:
Maybe fill in for 'blabla' here? :)

Reply:
Oh well, there was supposed to be a link to the other section there once I had made it :D